infra/cifuzz/external-actions/run_fuzzers/action.yml

```diff
@@ -40,10 +40,12 @@ inputs:
     description: |
       The branch of the git repo to use for storing certain artifacts from
       fuzzing.
+    default: main
```

Comment:
So now we are setting this by default instead of opting in. What are the consequences of this?